terraform/trial_rg/services.tf

```diff
@@ -62,20 +81,27 @@ module "trial_app_service_role" {
     "JDBC_DRIVER" = "com.microsoft.sqlserver.jdbc.SQLServerDriver"
     # TODO: replace with KeyVault reference
     "JDBC_URL"    = "jdbc:sqlserver://${module.roles_sql_server.sqlserver_name}.database.windows.net:1433;databaseName=ROLES;user=${module.roles_sql_server.db_user};password=${module.roles_sql_server.db_password}"
-    "WEBSITES_PORT"               = "8080" # The container is listening on 8080
+    "SPRING_PROFILES_ACTIVE"                  = var.spring_profile
+    "SPRING_CLOUD_CONFIG_LABEL"               = var.spring_config_label
+    "SERVER_PORT"                             = "80"
+    "WEBSITES_PORT"                           = "80"
+    "EUREKA_CLIENT_SERVICEURL_DEFAULTZONE"    = "${module.trial_sc_discovery.hostname}/eureka/"
+    "EUREKA_INSTANCE_HOSTNAME"                = "${local.role_name}.azurewebsites.net"
   }
 
   depends_on = [
     azurerm_app_service_plan.apps_service_plan,
     module.roles_sql_server,
+    module.trial_sc_config,
+    module.trial_sc_discovery,
   ]
 }
 
 # init service
 # todo: make 1-time service: ARTS-362
 module "trial_app_service_init" {
```

Comment:
In line 117 (Unable to comment there) why here on the depends_on there is no  module.trial_sc_config, module.trial_sc_discovery? The init is connected to the discovery...

Reply:
will add. the init is in a WIP status so I didn't change it...